packages/@aws-cdk/aws-dynamodb/lib/time-to-live-provider.ts

```diff
@@ -0,0 +1,67 @@
+import * as path from 'path';
+import * as lambda from '@aws-cdk/aws-lambda';
+import { Duration, Stack } from '@aws-cdk/core';
+import * as cr from '@aws-cdk/custom-resources';
+import { Construct } from 'constructs';
+
+// keep this import separate from other imports to reduce chance for merge conflicts with v2-main
+// eslint-disable-next-line no-duplicate-imports, import/order
+import { Construct as CoreConstruct } from '@aws-cdk/core';
+
+export class TimeToLiveProvider extends CoreConstruct {
+  /**
+   * Creates a stack-singleton resource provider.
+   */
+  public static getOrCreate(scope: Construct): TimeToLiveProvider {
+    const stack = Stack.of(scope);
+    const logicalId = '@aws-cdk/aws-dynamodb.TimeToLiveProvider';
+    return stack.node.tryFindChild(logicalId) as TimeToLiveProvider ?? new TimeToLiveProvider(stack, logicalId);
+  }
+
+  /**
+   * The custom resource provider.
+   */
+  public readonly provider: cr.Provider;
+
+  /**
+   * The onEvent handler
+   */
+  public readonly onEventHandler: lambda.Function;
+
+  /**
+   * The isComplete handler
+   */
+  public readonly isCompleteHandler: lambda.Function;
+
+  private constructor(scope: Construct, id: string) {
+    super(scope as CoreConstruct, id);
+
+    const code = lambda.Code.fromAsset(path.join(__dirname, 'table-time-to-live-handler'));
+
+    // Issues UpdateTable API calls
+    this.onEventHandler = new lambda.Function(this, 'OnEventHandler', {
+      code,
+      runtime: lambda.Runtime.NODEJS_12_X,
```

Comment:
We switched to Node 14 in the meantime 🙂.

Reply:
Alright, I have updated both to node 14.